### ISSUE TITLE
Improve UX to create an empty story pack

### ISSUE THREAD

**Issue by marian-m12l** [marian-m12l/studio#158]
I just started to use and test this wonderful application!
An impressive work, and a great tool to get full control on existing and custom stories. May thanks to the developers! :+1: 

**I haven't found how to create a new pack (from scratch)**.
There seems to be no button or link on neither of the two main pages (Pack library or Pack editor).
In case this button/link was just hidden, I checked the DOM, but no "create"-something on neither of the two pages.
Did I miss something?

I will continue testing the app. For now, I successfully:
- mounted my Lunii in STUdio ;
- downloaded the two packs from my Lunii to STUdio (the two packs being the official default pack, with metadata, and a custom story created on the Android app) ;
- open these two packs in the Pack editor, and test/simulate them in STUdio (listen and interact).

I haven't tried yet to add/remove nodes in a pack, nor upload a created/edited pack back to my Lunii.

My environment:
- Lunii v2 with firmware version 2.20
- STUdio 0.3.0-alpha.3-SNAPSHOT (cloned the repo and built few hours ago)
- (X)Ubuntu 20.04.1 (64 bits)
- Mozilla Firefox 84.0.1

_Originally posted by @MathieuIko in https://github.com/marian-m12l/studio/issues/97#issuecomment-756447535_

**Comment by marian-m12l** [comment]
@MathieuIko The intent was to show immediately to new users what a story pack diagram should look like, as it is probably not intuitive. You can create an empty story pack by clicking the "bin" button on top of the pack metadata. There's _definitely_ room for UX improvement here.

**Comment by MathieuIko** [comment]
Thank you for the tip!

Actually I first tried the Pack Editor by clicking "edit" on an existing pack, and switching between Pack Library and Pack Editor still showed the opened existing pack. Then I didn't found any button/link to create a new pack.
Finally I restarted the app, and came directly to the Pack Editor to see the sample pack.

This sample pack is a great idea!
It clearly helps understand the basics of a story pack creation.

If the user has already opened an existing pack, maybe a button/link to "close" this pack and "create" a new pack (maybe showing the sample pack) would be useful, for the user to be confident enough that he/she is creating a new pack and not editing/emptying the previously opened pack.
